Lab2/Lab2.ipynb

```diff
@@ -33,12 +33,12 @@
    "id": "4374bfb3-2ab5-4bb7-a7e2-4c07ecdb51c1",
    "metadata": {},
    "source": [
-    "In this lab, we will look at the use of convolutions and methods to convolve and use averages to filter signals in useful ways."
+    "In this lab, we will look at applying convolutions and averages to filter signals in useful ways. We will explore the Central Limit Theorem and the implications it has in real-world data, dealing with trials and detection thresholds. "
```

Comment:
General comment for entire lab - great job really making it read like a paper rather than discrete problems! Definitely a leap in the right direction on that front even just since lab 1! I need to catch up lol!